packages/devextreme/testing/testcafe/tests/editors/textArea/index.ts

```diff
@@ -1,101 +1,101 @@
+  .page(url(__dirname, '../../container.html'))
+  .afterEach(async () => clearTestPage());
 
 const text = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.';
+const longText = 'Prepare 2013 Marketing Plan: We need to double revenues in 2013 and our marketing strategy is going to be key here. R&D is improving existing products and creating new products so we can deliver great AV equipment to our customers.Robert, please make certain to create a PowerPoint presentation for the members of the executive team.';
+
+test('TextArea correct height with autoResizeEnabled, height is 7em & maxHeight is 5em', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await testScreenshot(t, takeScreenshot, 'TextArea correct height with autoResizeEnabled, height is 7em & maxHeight is 5em.png', { element: '#container' });
 
-[true, false].forEach((autoResizeEnabled) => {
-  test(`TextArea has correct height with "autoResizeEnabled" is ${autoResizeEnabled} and height is 7em & maxHeight is 5em`, async (t) => {
-    const textArea = new TextArea('#container');
-
-    await t
-      .expect(getPropertyValue(textArea.element, 'maxHeight'))
-      .eql('5em')
-      .expect(getPropertyValue(textArea.element, 'height'))
-      .eql('7em')
-      .expect(textArea.element.getStyleProperty('height'))
-      .eql('70px')
-      .expect(getPropertyValue(textArea.input, 'height'))
-      .eql('');
-  }).before(async () => createWidget('dxTextArea', {
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  await setAttribute('#container', 'style', 'width: 300px; height: 400px;');
+
+  const config = {
     maxHeight: '5em',
     height: '7em',
-    autoResizeEnabled,
-    width: 200,
-    value: text,
-  }));
-
-  test(`TextArea has correct height with "autoResizeEnabled" is ${autoResizeEnabled} and height is 5em & maxHeight is 7em`, async (t) => {
-    const textArea = new TextArea('#container');
-
-    await t
-      .expect(getPropertyValue(textArea.element, 'maxHeight'))
-      .eql('7em')
-      .expect(getPropertyValue(textArea.element, 'height'))
-      .eql('5em')
-      .expect(textArea.element.getStyleProperty('height'))
-      .eql('70px')
-      .expect(getPropertyValue(textArea.input, 'height'))
-      .eql('');
-  }).before(async () => createWidget('dxTextArea', {
-    maxHeight: '7em',
-    height: '5em',
-    autoResizeEnabled,
-    width: 200,
+    width: '100%',
     value: text,
-  }));
-
-  test(`TextArea has correct height with "autoResizeEnabled" is ${autoResizeEnabled} and maxHeight option is 5em`, async (t) => {
```

Comment:
We should not have deleted scenarios that were tested by these tests. We needed to preserve them, changing only the testting method to checking through screenshot testing.